src/main/events/library/open-game.ts

```diff
@@ -1,18 +1,52 @@
+import { exec as execCallback } from 'child_process';
 import { gameRepository } from "@main/repository";
-
 import { registerEvent } from "../register-event";
 import { shell } from "electron";
+import { promisify } from 'util';
+import os from 'os';
+
+const exec = promisify(execCallback);
+
+const escapeShellPath = (path: string) => {
+  if (os.platform() === 'win32') {
+    // Windows paths should be enclosed in quotes if they contain spaces
+    return `"${path}"`;
+  } else {
+    // Escapes spaces, parentheses for commands in Linux
+    return path.replace(/ /g, '\\ ').replace(/\(/g, '\\(').replace(/\)/g, '\\)');
+  }
+};
 
 const openGame = async (
   _event: Electron.IpcMainInvokeEvent,
   gameId: number,
   executablePath: string
 ) => {
-  await gameRepository.update({ id: gameId }, { executablePath });
+  try {
+    await gameRepository.update({ id: gameId }, { executablePath });
 
-  shell.openPath(executablePath);
+    // Check for Lutris only on Linux systems
+    if (os.platform() === 'linux') {
```

Comment:
You can use `process.platform` here, it'll generate less changes since you don't have to import the `os` and it'll work just as fine